### PR TITLE
Logo files with border and rendered name

### PR DESCRIPTION
**Pull request by thockin**
Move all log-related files to a logo/ dir.

Add a logo with the white border (common for dark backgrounds).

Add a rendered image of the name in the 'ubuntu' font.

Update usage guidelines:
- new links
- linewraps
- typos

### CODE REVIEW

**Comment by eparis** [comment]
can you add an OWNERS? so future PRs get an assignee? I don't even know who that would be....

**Comment by thockin** [comment]
Added OWNERS.  I am the owner.

File: logo_usage_guidelines.md [hunk 1]
```diff
@@ -1,18 +0,0 @@
-# Kubernetes Branding Guidelines
```
Comment (eparis) [inline logo_usage_guidelines.md]: You deleted this whole file, did you intend to `mv` it?

Comment (thockin) [inline logo_usage_guidelines.md]: I think I did mv it, I don;t know why it's not showing as such, but it's not that important.

**Pull request merged**